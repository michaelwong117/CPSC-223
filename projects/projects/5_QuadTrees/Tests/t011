#!/bin/bash
# pointset_nearest_neighbor for point not in set #1

trap "/usr/bin/killall -q -u $USER ./PointsetUnit 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./PointsetUnit ]; then
  echo './PointsetUnit is missing or not executable'
  echo './PointsetUnit is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./PointsetUnit 10 < /dev/null
