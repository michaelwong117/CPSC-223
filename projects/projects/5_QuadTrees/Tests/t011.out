Distance: 5.000000
Point: 8.000000 26.000000
